packages/amplify_api_rest_plugin_interface/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
```

Comment:
Please update

Reply:
Updated this basic log: 

`## 0.0.1-dev.5 (2020-12-20)

* Initial preview release of API plugin.
`